In [ ]:
# !pip install opencv-python
# !pip install ipykernelc

In [1]:
import cv2

In [2]:
# Read the image
image = cv2.imread("./mountain.jpg")

In [3]:
# check if image is loaded successfully

if image is None:
    print("Error : Cound not load the image.")
else:
    print("Image loaded successfully.")

Image loaded successfully.


In [5]:
# Display the image

cv2.imshow("Image window", image)
cv2.waitKey(0)
cv2.destroyAllWindows()

In [8]:
# Write the image

image_new = image + 30
cv2.imwrite("./output.jpg", image_new)


True

PRACTICE

In [2]:
import cv2

In [3]:
image=cv2.imread('handwritten.jpeg')

In [4]:
type(image)

numpy.ndarray

In [5]:
image.shape

(1293, 1134, 3)

#### 📐 Breakdown of Shape: (1293, 1134, 3)
##### Dimension	Meaning
- 1293	Height (number of rows / pixels vertically)
- 1134	Width (number of columns / pixels horizontally)
- 3	Color channels: Red, Green, Blue (RGB)

In [6]:
cv2.imshow(winname="Practice",mat=image)
cv2.waitKey(0)
cv2.destroyAllWindows()

In [7]:
image*=2
cv2.imwrite('testIm.jpeg',image)

True